hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/RawLocalFileSystem.java

```diff
@@ -425,7 +426,7 @@ public void completed(Integer result, Integer rangeIndex) {
           channel.read(buffer, range.getOffset() + buffer.position(), rangeIndex, this);
         } else {
           // Flip the buffer and declare success.
-          buffer.flip();
+          ((Buffer)(buffer)).flip();
```

Comment:
Do you use jdk17 build hadoop, and run on jdk8?
If so, you need check in all project like this, I think it has so many.